### ISSUE TITLE
Reopen last closed file

### ISSUE THREAD

**Issue by matchboxhero** [adobe/brackets#12015]
Hi,

Is it possible to bind a keyboard shortcut to reopen the last closed file, much like Sublime does with "Cmd+Shift+T"?

Thanks.

**Comment by petetnt** [comment]
@swmitra This sounds like a nice addition to #12012 IMO

**Comment by swmitra** [comment]
Yes @petetnt , this looks like a related stuff. But, there is a catch. In Brackets, we can have a full editor created for a document without adding it to working set ( view only ). There is no explicit workflow to close this editor unlike close button available on a working set entry. So, we have to define/assume what is actually a document close event means. Currently there are no such notifications. If we assume that a document is closed when it is removed from working set , then we already have this feature available as the closed document would be the first entry in MROF if there are no other document in working set , or it will be the second entry in MROF if there were other entries present in the working set and one them would be promoted as the first entry in the MROF.

**Comment by petetnt** [comment]
@swmitra Yeah, I was thinking about that: to me most sense would be an item that was added to the working set and then closed. I think that it would need another list, a Most Recently Closed List one. 

Opening a file would check if it was on the MRCL and then remove it from there (if it was). Closing a file would push it to the top of that list. This way you could `pop` files from the MRCL list until everything that you had open (not including the ones you have open now) would be open.
